src/pages/index.js

```diff
@@ -116,3 +211,70 @@ const addCardImageModal = new PopupWithForm(
   handleAddCardSubmit
 );
 addCardImageModal.setEventListeners();
+
+const modalPreviewImage = new PopupWithImage("#image-modal");
+modalPreviewImage.setEventListeners();
+
+/* -------------------------------------------------------------------------- */
+/*                           Button Event Listeners                           */
+/* -------------------------------------------------------------------------- */
+
+profileEditButton.addEventListener("click", () => {
+  const user = userInfo.getUserInfo();
+  profileTitleInput.value = user.name;
+  profileDescriptionInput.value = user.description;
+  modalEditForm.open();
+});
+
+addNewCardButton.addEventListener("click", () => {
+  addFormValidator._disableSubmitButton();
```

Comment:
`_disableSubmitButton`

Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.